AirSkyBoat/AirSkyBoat
# Fix several Enchantment and status effect gear issues

**_I affirm:_**


## Please enter a player-facing description
Players can now have several enchantment status effects active at the same time. Enchantment status effects also now wear off when zoning. Also Breath Mantle now works correctly. (Tracent)

## What does this pull request do? (Please be technical)
This PR fixes several issues:
- Fixes an exploit where players could keep an enchantment even after unequipping the gear that gives the enchantment
- Also allows multiple enchantment effects to be active on a player at the same time (was validated on retail by siknoz)
- Fixes breath mantle to actually give enchantment
- Fixes several pieces of gear that give the status effects that should not be wear off upon unequipping gear (based on JP wiki)
- Enchantment status effects now wear off when zoning

## Steps to test these changes
- Use breath mantle and check enchantment
- Use two enchantment items such as high breath mantle (15487) and vision ring (15559) to see two status effects and then remove two gear one by one to see only correct enchantment removed
- Message on discord for exploit testing instructions
- Use aero mufflers then remove and see that enaero status effect remains
- Zone with an enchantment to see it wear off

## Special Deployment Considerations
